pkg/cluster/provider.go

```diff
@@ -105,6 +106,27 @@ func (p *Provider) Delete(name, explicitKubeconfigPath string) error {
 	return internaldelete.Cluster(p.logger, p.ic(name), explicitKubeconfigPath)
 }
 
+// Deletes all clusters created by getting list of cluster names and using Delete function.
+func (p *Provider) DeleteAll(explicitKubeconfigPath string) error {
+	clusters, err := p.provider.ListClusters()
+	if err != nil {
+		return err
+	}
+
+	if len(clusters) == 0 {
+		return errors.New("No kind clusters found.")
```

Comment:
error strings should start lower cased.
the dot at the end is not needed.

`errors.New("no kind clusters found")`

Reply:
Fixed in latest push